weave-js/src/components/PagePanelComponents/Home/Browse3/pages/ChatView/hooks.ts

```diff
@@ -359,6 +82,22 @@ export const getChatFormat = (call: CallSchema): ChatFormat => {
   return ChatFormat.None;
 };
 
+export const normalizeChatCompletion = (
```

Comment:
The fact that we have to pass a request alongside the completion in order to normalize this indicates to me that either:
1. We should not have separate logic for handling these at all
2. We should omit the model from the completion

I think something like this makes sense:
```
type NormalizedChat {
  model: string, // Don't make this optional, just give it a placeholder like 'unknown' if we can't parse from anywhere
  request: ChatRequest,
  response: Omit<ChatResponse, 'model'>,
}
```

then just something like: 
```
const extractChatFromCall = (call: TraceCallSchema): NormalizedChat | null => {
  ...
}
 ```